modules/teacher.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+
+# top level comment on Teacher Class for linter error
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    @specialization = specialization
+    super(name, age, parent_permission)
```

Comment:
- [ ] Let's be consistent to avoid mixing up values. `age` should come before `name` here based on the Person class.

- [ ] The `parent_permission` uses the colon notation so the way of assigning it is somewhat different.

  ```
  super(age, name, parent_permission: parent_permission)
  ```